sdk/healthinsights/health-insights-radiologyinsights-rest/test/public/SampleAgeMismatchInferenceAsync.spec.ts

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { Context } from "mocha";
+import { AzureHealthInsightsClient, getLongRunningPoller } from "../../src";
+import { createClient, createRecorder } from "./utils/recordedClient";
+
+const codingData = {
+  system: "Http://hl7.org/fhir/ValueSet/cpt-all",
+  code: "USPELVIS",
+  display: "US PELVIS COMPLETE",
+};
+
+const code = {
+  coding: [codingData],
+};
+
+const patientInfo = {
+  sex: "female",
+  birthDate: new Date("1959-11-11T19:00:00+00:00"),
+};
+
+const encounterData = {
+  id: "encounterid1",
+  period: {
+    start: "2021-8-28T00:00:00",
+    end: "2021-8-28T00:00:00",
+  },
+  class: "inpatient",
+};
+
+const authorData = {
+  id: "authorid1",
+  fullName: "authorname1",
+};
+
+const orderedProceduresData = {
+  code: code,
+  description: "US PELVIS COMPLETE",
+};
+
+const administrativeMetadata = {
+  orderedProcedures: [orderedProceduresData],
+  encounterId: "encounterid1",
+};
+
+const content = {
+  sourceType: "inline",
+  value:
+    "CLINICAL HISTORY:   " +
```

Comment:
Curious about `+` all these lines.  Are these test generated? Could we use backticks for multiline strings?
```ts
  value: `CLINICAL HISTORY:
20-year-old female presenting with abdominal pain. Surgical history significant for appendectomy.

COMPARISON:
// omitted

`,
```